Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG VERSION=lts
+ARG VERSION=18
```

Comment:
Is this hardcoded because LTS is now 20?

Reply:
yes, this is just for the fallback. All uses currently use the version specified in `.nvmrc`.

Reply:
if it's a fallback why not leave it as `lts`?

Reply:
because node 20 fails.

Reply:
can you provide more info? i have been testing node 20 and haven't find any issues so far, could help me find any problems

Reply:
you will need to run the integration tests with node 20. This also locks the version used to a node version which is supported. LTS changes ideally this should not end up in a broken state each time a new version becomes LTS

Reply:
thanks for the context. Yeah that's true, better to be safe by locking the version and upgrade as soon as we test newer versions